frontend/src/api/k8s/__tests__/inferenceServices.spec.ts

```diff
@@ -155,3 +219,256 @@ describe('assembleInferenceService', () => {
     expect(inferenceService.spec.predictor.minReplicas).toBeUndefined();
   });
 });
+
+describe('listInferenceService', () => {
+  it('should return list of inference service', async () => {
+    const inferenceServiceMock = mockInferenceServiceK8sResource({});
+    k8sListResourceMock.mockResolvedValue(mockK8sResourceList([inferenceServiceMock]));
+    const result = await listInferenceService();
+    expect(result).toStrictEqual([inferenceServiceMock]);
+    expect(k8sListResourceMock).toHaveBeenCalledTimes(1);
+    expect(k8sListResourceMock).toHaveBeenCalledWith({
+      model: {
+        apiGroup: 'serving.kserve.io',
+        apiVersion: 'v1beta1',
+        kind: 'InferenceService',
+        plural: 'inferenceservices',
+      },
+      queryOptions: {},
+    });
+  });
+
+  it('should handle errors and rethrows', async () => {
+    k8sListResourceMock.mockRejectedValue(new Error('error'));
+    await expect(listInferenceService()).rejects.toThrow('error');
+    expect(k8sListResourceMock).toHaveBeenCalledTimes(1);
+    expect(k8sListResourceMock).toHaveBeenCalledWith({
+      model: {
+        apiGroup: 'serving.kserve.io',
+        apiVersion: 'v1beta1',
+        kind: 'InferenceService',
+        plural: 'inferenceservices',
+      },
+      queryOptions: {},
+    });
+  });
+});
+describe('listScopedInferenceService', () => {
+  it('should return list of scoped inference service', async () => {
+    const inferenceServiceMock = mockInferenceServiceK8sResource({});
+    const { name } = inferenceServiceMock.metadata;
+    k8sListResourceMock.mockResolvedValue(mockK8sResourceList([inferenceServiceMock]));
+    const result = await listScopedInferenceService();
+    expect(result).toStrictEqual([inferenceServiceMock]);
+    expect(k8sListResourceMock).toHaveBeenCalledWith({
```

Comment:
If we're expecting two calls, there should be two mocks. Projects and inference services.